apps/cms/src/lib/collection-service.ts

```
@@ -0,0 +1,63 @@
+
+  // Delete a collection
+  static async deleteCollection(slug: string): Promise<void> {
+    // Simulate API call
+    return new Promise((resolve) => {
+      setTimeout(() => {
+        this.collections = this.collections.filter((c) => c.slug !== slug);
+        resolve();
+      }, 500);
+    });
+  }
+}
```

Biome (1.9.4)

[error] Avoid classes that contain only static members.

Prefer using simple functions instead of classes with only static members.

(lint/complexity/noStaticOnlyClass)